src/main/groovy/nebula/plugin/dependencylock/ConfigurationsToLockFinder.groovy

```diff
@@ -0,0 +1,76 @@
+/**
+ *
+ *  Copyright 2018 Netflix, Inc.
```

Comment:
not a big deal but this might have to be 2019  :)

Reply:
Haha, yes! I'll adjust that. Thanks!